tendermint/src/rpc/id.rs

```diff
@@ -3,9 +3,17 @@
 use getrandom::getrandom;
 use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq, Ord, PartialOrd)]
+#[serde(untagged)]
+pub enum IdType {
```

Comment:
Does this internal type need to be public? Also, I'm surprised the linter doesn't complain about missing docs here.

Reply:
This is now obsolete, because IdType was removed and Id was made into an enum.